src/target/semihosting.c

```diff
@@ -856,9 +870,13 @@ int32_t semihosting_handle_request(target_s *const target, const semihosting_s *
 	case SEMIHOSTING_SYS_TMPNAM:
 		return semihosting_temp_name(target, request);
 
+	case SEMIHOSTING_SYS_ELAPSED:
+		return semihosting_elapsed(target, request);
+
+	case SEMIHOSTING_SYS_TICKFREQ:
+		/* 1000 Hz SysTick, or BMDA "precision". Servicing breakpoints over SWD is not fast. */
+		return SYSTICKHZ;
 	// not implemented yet:
```

Comment:
Given this replaces the only reason for this comment, please remove the comment as it's not useful any more.